### PR TITLE
Cross-mio updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Added cruises:
ASMAT2a & 2b test glider cruises (QLRs 3213-00009 & -00010)
AST3 recovery / AST4 deployment cruise (confirmed start/end date/time with Sheri per Chief Sci notebook - start 2023-10-23T14:29:00; end 2023-11-04T13:11:00)
Moved Papa 10a glider cruise so it was in correct chronological order

Sensor bulk load:
Added 3 CTDBP-C sensors (converted from CTDBP-Es to -Cs for Pioneer MAB).  Can verify via WHOI_Asset_Tracking.xlsx in Vault.

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -331,5 +331,8 @@ TN422,Thomas G. Thompson,2023-08-13T00:00:00,2023-09-18T23:59:59,Cabled (rvdata)
 EK20230823,Elakha,2023-08-23T16:19:00,2023-08-23T20:09:00,Endurance cspp deployment CE01ISSP-00022
 AR76-03,Neil Armstrong,2023-08-28T10:00:00,2023-09-21T12:08:00,Irminger 10
 AT50-16,Atlantis,2023-09-21T15:40:00,2023-10-07T17:00:00,Endurance 19 moorings profilers and gliders
+TS20231004,Tenacious II,2023-10-04T12:00:00,2023-10-05T04:00:00,ASMAT2a test glider deployments
+SU20231023,Pacific Surveyor,2023-10-23T14:00:00,2023-10-24T12:00:00,Papa 10a glider deployments
+AR78,Neil Armstrong,2023-10-23T14:29:00,2023-11-04T13:11:00,AST3 PM anchor and SM recoveries and AST4 SWM deployment
 EK20231026,Elakha,2023-10-26T16:01,2023-10-26T20:48:00,Endurance glider deployment 384-00014
-SU20231023,Pacific Surveyor,2023-10-23T14:00:00,2023-10-24T12:00:00,Papa 10a glider deployments
\ No newline at end of file
+TS20231106,Tenacious II,2023-11-06T04:00:00,2023-11-07T02:00:00,ASMAT2b test glider deployments
```
Comment (cdobs) [inline cruise/CruiseInformation.csv:338]: looks good

**Review by cdobs — APPROVED** [review]
looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -563,8 +564,10 @@ CGINS-CTDBPC-50015,A01034,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C (C-
...
 CGINS-CTDBPC-50056,A01089,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C (C-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-50056,,20150604,11835,,
 CGINS-CTDBPC-50057,A01146,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C (C-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-50057,,20141027,11835,,
+CGINS-CTDBPC-50067,A01091,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C (C-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-50067,,20150604,11835,converted from CTDBP-E to -C for Pioneer MAB,Woods Hole Oceanographic Institution
 CGINS-CTDBPC-50108,A01333,Sensor,0,CTD Pumped: CTDBP Series C,CTDPB SERIES C,Sea-Bird Electronics,SBE 16plus V2,16-50108,,20150410,13535,,
 CGINS-CTDBPC-50109,A01334,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C,Sea-Bird Electronics,SBE 16plus V2,16-50109,,20150414,13525,,
+CGINS-CTDBPC-50111,A01336,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C (C-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-50111,,20150414,13535,converted from CTDBP-E to -C for Pioneer MAB,Woods Hole Oceanographic Institution
```
Comment (cooleyky) [inline bulk/sensor_bulk_load-AssetRecord.csv:570]: Looks good

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (cooleyky) [inline cruise/CruiseInformation.csv:338]: looks good

**Review by cooleyky — APPROVED** [review]
Updates look good

**Pull request merged**